homeassistant/components/squeezebox/media_player.py

```diff
@@ -544,5 +544,5 @@ def async_call_method(self, command, parameters=None):
         all_params = [command]
         if parameters:
             for parameter in parameters:
-                all_params.append(urllib.parse.quote(parameter, safe="+:=/?"))
+                all_params.append(parameter)
```

Comment:
The comment still describes that it escapes it.

Reply:
Whoops